website/conferences/model.py

```diff
@@ -19,6 +18,9 @@ class Conference(StoredObject):
     name = fields.StringField(required=True)
     info_url = fields.StringField(required=False, default=None)
     logo_url = fields.StringField(required=False, default=None)
+    location = fields.StringField(required=False, default=None)
+    start_date = fields.DateTimeField(default=None)
+    end_date = fields.DateTimeField(default=None)
     active = fields.BooleanField(required=True)
```

Comment:
Do we care to remove this field in the database? What's the requirement, I wonder?